pkg/ccl/backupccl/targets.go

```diff
@@ -433,7 +393,23 @@ func selectTargets(
 	allDescs, lastBackupManifest := loadSQLDescsFromBackupsAtTime(backupManifests, asOf)
 
 	if descriptorCoverage == tree.AllDescriptors {
-		return fullClusterTargetsRestore(allDescs, lastBackupManifest)
+		fullClusterDescs, fullClusterDBs, err := fullClusterTargets(allDescs)
```

Comment:
nit: can we add this back as a method please, it helps differentiate full cluster restore logic from non-cluster restore logic.